app/assets/javascripts/blacklight/facet_load.js

```diff
@@ -9,11 +9,11 @@
 
     $('ul.facet-values, ul.pivot-facet').each(function(){
       var longest = $(this).find('span.facet-count').sort(longer).first();
-      var clone = longest.clone()
-        .css('visibility','hidden').css('width', 'auto');
-      $('body').append(clone);
-      $(this).find('.facet-count').first().width(clone.width());
-      clone.remove();
+      
+      if (longest && longest.textContent) {
```

Comment:
`textContent` is the native API I believe?  So we would need either `longest[0].textContent` or use jQuery's `longest.text()`?